kubernetes/catalog/kubernetes/kubernetes.bom

```diff
@@ -1,11 +1,12 @@
 brooklyn.catalog:
   version: "2.1.0-SNAPSHOT" # CLOCKER_VERSION
 
+  brooklyn.libraries:
+  - mvn:io.brooklyn.clocker/clocker-common
+  - mvn:io.brooklyn.etcd/brooklyn-etcd
+
   items:
-  - classpath://io.brooklyn.etcd.brooklyn-etcd:etcd/etcd.bom
-  - classpath://io.brooklyn.clocker.common:docker/docker.bom
-  - classpath://io.brooklyn.clocker.common:common/haproxy.bom
-  - classpath://io.brooklyn.clocker.kubernetes:kubernetes/brooklyn-dns-etc-hosts-generator.bom
+  - classpath://kubernetes/brooklyn-dns-etc-hosts-generator.bom
```

Comment:
The way this bundle makes it way into clocker seems hacky now https://github.com/brooklyncentral/clocker/blob/master/kubernetes/pom.xml#L39-L56

Can you update it as part of this PR please